### ISSUE TITLE
Getting TypeError on reports processing

### ISSUE THREAD

**Issue by macrotex** [furlongm/patchman#386]
Seeing this error when running `patchman -a -q`:
```
Traceback (most recent call last):
  File "/usr/bin/patchman", line 687, in <module>
    main()
  File "/usr/bin/patchman", line 681, in main
    showhelp = process_args(args)
  File "/usr/bin/patchman", line 655, in process_args
    host_updates_alt(args.host)
  File "/usr/bin/patchman", line 315, in host_updates_alt
    phost.updates = updates
  File "/usr/lib/python3/dist-packages/django/db/models/fields/related_descriptors.py", line 536, in __set__
    raise TypeError(
TypeError: Direct assignment to the forward side of a many-to-many set is prohibited. Use updates.set() instead.
```
I am running patchman 2.0.1-1.

**Comment by furlongm** [comment]
This was fixed in #357

I'll try to get a release out this week

**Comment by macrotex** [comment]
I think this is fixed in 2.0.2(?).

**Comment by furlongm** [comment]
Fixed in 2.0.3
